### PR TITLE
Update bursty_traffic_test.go

### PR DESCRIPTION
**Pull request by saarvind6481**
cisco-dp-1.11-uprev
Below are  resaons that has  brought about changes in PR from Cisco 

1] We have implemented recent yang uprev that has added queue-id to the queue openconfig Model

2] We have Modified classifier requirement in line with Openconfig requirement to have different policies for ipv4 ad ipv6

3] we have removed  forwarding-group implementation , which earlier had to be commited along with the classifier.

4] Implemented dropEnabled leaf

### CODE REVIEW

**Review by prinikasn — CHANGES_REQUESTED** [review]
please also update the otg_tests/ files for this test.

File: feature/qos/ate_tests/bursty_traffic_test/bursty_traffic_test.go [hunk 1]
```diff
@@ -400,9 +392,9 @@ func TestBurstyTraffic(t *testing.T) {
 				counters["dutQosOctetsBeforeTraffic"][data.queue] = gnmi.Get(t, dut, gnmi.OC().Qos().Interface(dp3.Name()).Output().Queue(data.queue).TransmitOctets().State())
 				counters["dutQosDroppedPktsBeforeTraffic"][data.queue] = gnmi.Get(t, dut, gnmi.OC().Qos().Interface(dp3.Name()).Output().Queue(data.queue).DroppedPkts().State())
 				//counters["dutQosDroppedOctetsBeforeTraffic"][data.queue] = gnmi.Get(t, dut, gnmi.OC().Qos().Interface(dp3.Name()).Output().Queue(data.queue).DroppedOctets().State())
-				if !deviations.QOSDroppedOctets(dut) {
```
Comment (prinikasn) [inline feature/qos/ate_tests/bursty_traffic_test/bursty_traffic_test.go:403]: This deviation is currently used by other vendors too, so we can remove them yet.

Comment (saarvind6481) [reply feature/qos/ate_tests/bursty_traffic_test/bursty_traffic_test.go:403]: This is in cisco specific config code, where we are not using the deviations, as we support Dropped Octets now
So this should not affect other vendors.

Comment (saarvind6481) [reply feature/qos/ate_tests/bursty_traffic_test/bursty_traffic_test.go:403]: Added back the deviations

Comment (saarvind6481) [reply feature/qos/ate_tests/bursty_traffic_test/bursty_traffic_test.go:403]: Also added the OTG tests

**Review by prinikasn — APPROVED** [review]
/gcbrun

**Pull request merged**